index.js

```diff
@@ -198,7 +199,7 @@ class WritableState {
   updateNextTick () {
     if ((this.stream._duplexState & WRITE_NEXT_TICK) !== 0) return
     this.stream._duplexState |= WRITE_NEXT_TICK
-    process.nextTick(updateWriteNT, this)
+    queueMicrotask(updateWriteNT.bind(this, this))
```

Comment:
We should just bind this once in the constructor similar to how we do it with afterRead (and change updateWriteNT to use this like afterRead does.